test/src/test/resources/fixtures/namespaced/one-of-the-namespaced-services-returns-error-reversed.yml

```diff
@@ -0,0 +1,202 @@
+      issue: IssueQuery
+    }
+
+    type IssueQuery {
+      search: SearchResult
+    }
+
+    type SearchResult {
+      id: ID
+      count: Int
+    }
+query: |
+  {
+    issue {
+      getIssue {
+        text
+      }
+      search {
+        count
+      }
+    }
+  }
+variables: {}
+serviceCalls:
+  current:
+    - serviceName: Issues
+      request:
+        query: |
+          query nadel_2_Issues {
+            issue {
+              getIssue {
+                text
+              }
+            }
+          }
+        variables: {}
+        operationName: nadel_2_Issues
+      # language=JSON
+      response: |-
+        {
+          "data": {
+            "issue": null
+          },
+          "errors": [
+            {
+              "message": "Error on IssueSearch"
+            }
+          ],
+          "extensions": {}
+        }
+    - serviceName: IssueSearch
+      request:
+        query: |
+          query nadel_2_IssueSearch {
+            issue {
+              search {
+                count
+              }
+            }
+          }
+        variables: {}
+        operationName: nadel_2_IssueSearch
+      # language=JSON
+      response: |-
+        {
+          "data": {
+            "issue": {
+              "search": {
+                "count": 1
+              }
+            }
+          },
+          "extensions": {}
+        }
+  nextgen:
+    - serviceName: Issues
+      request:
+        query: |
+          query {
+            ... on Query {
+              issue {
+                ... on IssueQuery {
+                  getIssue {
+                    ... on Issue {
+                      text
+                    }
+                  }
+                }
+              }
+            }
+          }
+        variables: {}
+        operationName: null
+      # language=JSON
+      response: |-
+        {
+          "data": {
+            "issue": null
+          },
+          "errors": [
+            {
+              "message": "Error on IssueSearch"
+            }
+          ],
+          "extensions": {}
+        }
+    - serviceName: IssueSearch
+      request:
+        query: |
+          query {
+            ... on Query {
+              issue {
+                ... on IssueQuery {
+                  search {
+                    ... on SearchResult {
+                      count
+                    }
+                  }
+                }
+              }
+            }
+          }
+        variables: {}
+        operationName: null
+      # language=JSON
+      response: |-
+        {
+          "data": {
+            "issue": {
+              "search": {
+                "count": 1
+              }
+            }
+          },
+          "extensions": {}
+        }
+# language=JSON
+response: |-
+  {
+    "data": {
+      "issue": {
+        "search": {
+          "count": 1
+        }
+      }
+    },
+    "errors": [
+      {
+        "message": "Error on IssueSearch",
+        "locations": [],
```

Comment:
I wonder if `locations` here should point to `data/issue` or `data/issue/getIssue`. There's very little detail about this error in the response, with a large query this could be a bit confusing to consumers.
Just a thought. I don't think this is in the scope of this PR.

Reply:
I mean the error is just forwarded from the underlying service. We have a `locations` only because we parse it into `GraphQLError` then back into `Map` -> `String`.

If an underlying service returns N GraphQL errors, it'll be hard to figure out what belongs where.